src/TurbulenceConvectionUtils.jl

```diff
@@ -240,28 +220,16 @@ function run_reference_SCM(
     namelist_args = nothing,
 )
     output_dir = scm_dir(m)
-    if ~isdir(output_dir) | overwrite
-        namelist = get_scm_namelist(m, overwrite = overwrite)
-        # Set optional namelist args
-        namelist["stats_io"]["calibrate_io"] = true
-        if !isnothing(namelist_args)
-            for namelist_arg in namelist_args
-                change_entry!(namelist, namelist_arg)
-            end
-        end
+    if ~isdir(joinpath(output_dir, "stats")) | overwrite
+        namelist = get_scm_namelist(m)
```

Comment:
```suggestion
        namelist = get_scm_namelist(m; overwrite = true)
```
Since overwrite defaults to `false`

Reply:
Actually, ` get_scm_namelist(m)` is just a deep copy now, it doesn't regenerate the namelist. I have now changed all the namelist overwrite defaults to true since we do not need to run TC.jl anymore.

The overwrite in this function refers more to overwrite of the `stats` directory by running TC.jl. I have changed the docstring accordingly.